base/dft.jl

```diff
@@ -580,7 +580,7 @@ module FFTW
     """
     function plan_r2r end
 
-    Base.USE_GPL_LIBS && include(joinpath("fft", "FFTW.jl"))
+    (Base.USE_GPL_LIBS || Base.USE_INTEL_MKL_FFT) && include(joinpath("fft", "FFTW.jl"))
```

Comment:
Shouldn't we just make `fftw_vendor()` return false if there isn't one and then include the FFTW interface file as long as this function does not return false?